apps/wing/src/cli.ts

```diff
@@ -235,6 +236,12 @@ async function main() {
         .preset(3)
         .argParser(parseInt)
     )
+    .addOption(
+      new Option(
```

Comment:
I think the default value can be provided as an argument to this constructor as well